### PR TITLE
ARXIVNG-1804 ability to cancel withdrawal and cross-list requests

### PR DESCRIPTION
**Pull request by erickpeirson**
Tweaked the slug when there is an open cross-list or withdrawal request:

image

Clicking the trash-can asks for confirmation:

image

et voila!

image

### CODE REVIEW

**Comment by erickpeirson** [comment]
Oh, I also got rid of the word "publish" just about everywhere. NBD.

**Comment by erickpeirson** [comment]
This depends on https://github.com/arXiv/arxiv-submission-core/pull/59

File: submit/templates/submit/confirm_delete_submission.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
             uploaded for this submission from your account. Are you sure you
             want to delete?
           {% else %}
-            Deleting will revert your article to the most recently published
+            Deleting will revert your article to the most recently announced
             and announced version and discard any new information entered or
```
Comment (eawoods) [inline submit/templates/submit/confirm_delete_submission.html]: announced and announced

Comment (erickpeirson) [reply submit/templates/submit/confirm_delete_submission.html]: much announced

Comment (erickpeirson) [reply submit/templates/submit/confirm_delete_submission.html]: fixed

**Pull request merged**